.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+    target-branch: "next"
```

Comment:
Do we want to create pull requests against the `next` branch? If not, this line should be removed

Reply:
@AleksandarDev please resolve this.

Reply:
@radovix Should be resolved now